modules/overviews/app/components/overviews/page_header_component.html.erb

```diff
@@ -48,19 +48,21 @@
         tab_nav.with_tab(
           test_selector: "project-overview-tab",
           selected: current_page?(project_overview_path),
-          href: helpers.project_overview_path,
+          href: helpers.project_overview_path
         ) do |t|
           t.with_icon(icon: :"op-view-split")
           t.with_text { I18n.t("overviews.label_overview") }
         end
 
-        tab_nav.with_tab(
-          test_selector: "project-dashboard-tab",
-          selected: current_page?(dashboard_project_overview_path),
-          href: helpers.dashboard_project_overview_path,
-        ) do |t|
-          t.with_icon(icon: :"op-view-list")
-          t.with_text { I18n.t("overviews.label_dashboard") }
+        if current_user.allowed_in_project?(:view_members, project)
```

Comment:
I don't know what was discussed as the linked meeting does not really reveal that. However, just looking at the code, it makes no sense to hide the whole dashboard when a user does not have the permission to view members. We don't even know, whether the members widget is shown in the dashboard. Maybe I am missing something here, but this looks wrong.